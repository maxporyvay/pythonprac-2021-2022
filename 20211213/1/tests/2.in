[21, 77, 888, 21, 45, 6, -90, 4, 76, 88, 2, 1, 3, 4, 5, 24]