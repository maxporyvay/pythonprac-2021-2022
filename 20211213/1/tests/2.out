[-90, 1, 2, 3, 4, 4, 5, 6, 21, 21, 24, 45, 76, 77, 88, 888]
